src/items.rs

```diff
@@ -225,10 +227,7 @@ impl<'a> FmtVisitor<'a> {
         }
 
         // A conservative estimation, to goal is to be over all parens in generics
-        let args_start = generics.ty_params
-                                 .last()
-                                 .map(|tp| end_typaram(tp))
-                                 .unwrap_or(span.lo);
+        let args_start = generics.ty_params.last().map(|tp| end_typaram(tp)).unwrap_or(span.lo);
```

Comment:
We should probably check if the source code has linebreaks, and if so, to stick with a multi-line chain.